yb-voyager/src/callhome/diagnostics.go

```diff
@@ -102,6 +102,11 @@ type UnsupportedFeature struct {
 	TotalOccurrences int      `json:"TotalOccurrences"`
 }
 
+type UnsupportedQueryConstruct struct {
```

Comment:
Let's discuss this in standup. I think we may need to think a little more and standardize the callhome json structure for assessment

Reply:
We can flatten it as we do for ybd. Not sure if it would break anything or not.